ui/shared/tabs/failureSummary/FailureSummaryTab.jsx

```diff
@@ -153,6 +154,18 @@ class FailureSummaryTab extends React.Component {
 
     return (
       <div className="w-100 h-100" role="region" aria-label="Failure Summary">
+        {!!suggestions.length && (
+          <a href="/push-health">
+            <Button
+              outline
+              className="bg-light failure-summary-push-health-button"
+              type="button"
+            >
+              View In Push Health
```

Comment:
I wonder if we should try to get people to go to push health before they click on a job. I recall that having the different apps in the upper left dropdown menu "Treeherder" icon isn't obvious so what do you think about adding a "Debug with push health" link in the top right menu - next to the notification icon?

Reply:
Also this button is showing up in the Push Health failure summary tab too :)

Reply:
Would this be in the same row as the `In Progress`, `N items` button that already goes to Push Health?

Reply:
I was thinking in the top navigation menu, next to the little bell icon. What do you think?

![Screen Shot 2021-01-28 at 1 38 55 PM](https://user-images.githubusercontent.com/19615783/106202098-45c88c80-616e-11eb-9025-1a1eb0d9f419.png)

Reply:
Ah, right! Yes, can do. Do you want to move the Perfherder and Intermittent links out, too?

Reply:
Here's what it looks like: 
![image](https://user-images.githubusercontent.com/49649/106294049-b201c980-621c-11eb-9604-87155dcd701f.png)

I'm OK with it if you are, but seeing it makes me wonder if it's confusing to have a `Push Health` and `Health` icon up there.

Reply:
Hah, that's a really good point. I doubt anyone knows what that health dropdown menu is for (I had to look). It basically determines if the little progress/failure tags (linking to push health) for each push get shown on try or all repos. I think we should default it to all and remove that menu option entirely.

Reply:
> Ah, right! Yes, can do. Do you want to move the Perfherder and Intermittent links out, too?

Sorry, I missed this. Lets leave those because I think that's one of the only ways to get to those views without manually changing the URL.

Reply:
I realized I should probably sanity check the idea of removing the 'health' dropdown menu in TH and defaulting to Try. @jmaher Does this sound reasonable or do you think 'all' would be a better default?

Reply:
I also wonder if defaulting to `All` might be better, since it would make sure we don't remove a useful feature anywhere. (I imagine there's a good reason we didn't make it the default already, but I don't know what that is...)

Reply:
this was added for perf reasons as the code sheriffs found poor performance across large revision ranges and the all | try | none was a method to toggle where to add push health data.  I think performance has improved.  either check with the sheriffs or be prepared to adjust if there are problems.

Reply:
@Archaeopteryx Could you give some input on this change please?

Reply:
No performance issues noticed with Push Health active for autoland and central.

Reply:
Ok, @bhearsum if you want to change that to 'all' that's the last thing to do. Feel free to merge when its ready.